local/include/core/http3.h

```diff
@@ -297,7 +297,10 @@ class H3Session : public Session
 
   /** Establish a QUIC connection from the given interface to the given IP
    * address. */
-  swoc::Errata do_connect(swoc::TextView interface, swoc::IPEndpoint const *target) override;
+  swoc::Errata do_connect(
+      swoc::TextView interface,
+      swoc::IPEndpoint const *target,
+      ProxyProtocolVersion pp_version = ProxyProtocolVersion::NONE) override;
```

Comment:
Let's add a @note comment explaining that while the pp_version is added to fit the override signature, there is no current support for Proxy Protocol for QUIC connections.

Reply:
Good point. Done.